server/dash/dashboards/neighborhood_recent.py

```diff
@@ -174,6 +174,7 @@ def update_table(selected_council):
     Input("council_list", "value")
 )
 def update_text(selected_council):
+    print(selected_council)
```

Comment:
If we want to log this in prod, let's try to make this as descriptive as possible, e.g.,
`print("neighborhood_recent.update_text: selected_council is ", selected_council)`

Reply:
Thanks for the catch - removing print statement for testing purposes.